### ISSUE TITLE
[pixie-chart] Can't apply custom memory limit/request on vizier-pem

### ISSUE THREAD

**Issue by rtrindvg** [newrelic/helm-charts#828]
**Bug description**

I can't seem to find the proper way to define the memory limit/request for the pixie-chart "vizier-pem" pods/daemonset.

**Version of Helm and Kubernetes**

helm v3.8.1
k8s v1.22.6

**Which chart?**

nri-bundle v4.4.6

**What happened?**

I am not sure this is a bug, but I have no clue if there is a proper channel for asking these before reporting here. I am Pulumi IaC to deploy nri-bundle to my AKS cluster. So far so good, but the vizier-pem pods are requesting 2Gi of memory, and I want to reduce this. Since this is a chart outside of this repo, I am having trouble identifying how can I pass this value properly through the configuration. I tried setting this using "pixie-chart.pemMemoryLimit" and "pixie-chart.pemMemoryRequest", which is accepted by the configuration and the apply passes the value forward, but the pods continue using the previous default value of 2Gi, and the daemon-set continues with the same definition as before.

So, in summary, I need help understanding if this is a bug or I just not have a proper configuration to make the change reach the daemonset definition.

**What you expected to happen?**

The daemonset should accept the new values.

**How to reproduce it?**

Apply a nri-bundle with pixie-chart enabled. Later try to change the memory values.

**Anything else we need to know?**

Since I am using Pulumi, please do not suggest directly kubectl commands to apply configuration changes, since this won't work on my CI/CD workflows. I need to pass this on the nri-bundle chart configuration options.
If this is not the proper channel for these kind of questions (which are maybe not bugs), kindly let me know so I can follow up this elsewhere.

**Comment by maur1th** [comment]
Same issue here. It's especially an issue on non production environments as we work with smaller instance sizes there.
